llama.cpp

```diff
@@ -4396,6 +4437,9 @@ static void llm_load_vocab(
         } else {
             if (tokenizer_model == "gpt2") {
                 vocab.type = LLAMA_VOCAB_TYPE_BPE;
+                if (model.arch == LLM_ARCH_LLAMA) {
+                    vocab.add_space_prefix = false;
+                }
```

Comment:
Is this needed - looks wrong?

Reply:
sorry, it should be `LLM_ARCH_GRANITE_SMALL`